docs/en/observability/monitor-k8s/monitor-k8s-application-performance.asciidoc

```diff
@@ -1,6 +1,6 @@
 [discrete]
 [[monitor-kubernetes-application-performance]]
-== Part 3: Monitor application performance
+== Part 4: Monitor application performance
 
 Quickly triage and troubleshoot application performance problems with the help of Elastic
 application performance monitoring (APM).
```

Comment:
L12 - L23

We should change this to "Set up APM". We don't use the term APM Server in ESS. In ESS, it's called the "Integrations Server."